src/bob/api/health_check.clj

```diff
@@ -15,28 +15,76 @@
 
 (ns bob.api.health-check
   (:require [clojure.java.io :as io]
+            [clojure.core.async :as a]
             [hugsql.core :as sql]
             [clj-docker-client.core :as docker]
             [manifold.deferred :as d]
             [failjure.core :as f]
             [taoensso.timbre :as log]
-            [bob.util :as u]
+            [aleph.http :as http]
+            [chime :refer [chime-ch]]
+            [clj-time.core :as t]
+            [clj-time.periodic :as tp]
+            [bob.artifact.db :refer [get-artifact-stores]]
+            [bob.resource.db :refer [get-external-resources]]
             [bob.states :as states]))
 
 (sql/def-db-fns (io/resource "sql/health_checks.sql"))
 
+(defn ping-external-systems []
+  (let [external-systems (concat (get-artifact-stores states/db) (get-external-resources states/db))]
+    (map #(when (f/failed? (f/try* @(http/get (clojure.string/join "/" [(:url %) "ping"]) {:throw-exceptions false}))) (:name %))
```

Comment:
Remove the `{:throw-exceptions false}` to let it blow up and the `f/try*` will catch it for a proper failjure.